src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1586,8 +1553,6 @@ public function getPreferredFormat(?string $default = 'html'): ?string
     /**
      * Returns the preferred language.
      *
-     * @param array $locales An array of ordered available locales
```

Comment:
I think we should document this as `@param string[] $locales`. otherwise the return type would also not work.